data/json/furniture.json

```diff
@@ -3375,6 +3375,25 @@
       ]
     }
   },
+  {
+    "id": "f_firering",
+    "type": "furniture",
+    "name": "fire ring",
+    "symbol": "#",
+    "bgcolor": [ "white" ],
+    "move_cost_mod": 2,
+    "required_str": -1,
+    "bash": {
+      "str_min": 15,
+      "str_max": 105,
+      "sound": "crash!",
+      "sound_fail": "whump!",
+      "items": [ { "item": "rock", "count": [ 5, 15 ] } ]
+    },
+    "deconstruct": { "items": [ { "item": "rock", "count": 20 } ] },
+    "flags": [ "TRANSPARENT", "CONTAINER", "FIRE_CONTAINER", "SUPPRESS_SMOKE", "PLACE_ITEM", "EASY_DECONSTRUCT" ],
```

Comment:
I believe `SUPPRESS_SMOKE` flag is wrong here. How it's supposed to suppress smoke if it's just essentially a bunch of rocks?

Reply:
good point, removed.